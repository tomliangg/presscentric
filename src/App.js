import React from 'react'
import AccountManager from './AccountManager'


const App = () => (
  <AccountManager />
)

export default App;
