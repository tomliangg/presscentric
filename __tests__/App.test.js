import React from 'react'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import App from '../src/App'


describe('<App />', () => {
  it('should render App', () => {
    const wrapper = shallow(<App />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})