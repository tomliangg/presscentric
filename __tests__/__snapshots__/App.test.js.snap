// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render App 1`] = `<AccountManager />`;
